va/va.h

```diff
@@ -381,6 +381,25 @@ typedef enum
      * (VAEncFEIMVBufferType, VAEncFEIMBModeBufferType) are needed for PAK input.
      **/
     VAEntrypointFEI         = 11,
+    /**
+     * \brief VAEntrypointStats
+     *
+     * Statistics for FEI and non-FEI. A pre-processing function for getting some
+     * statistics and motion vectors is added, and some extra controls for Encode
+     * pipeline are provided. The application can optionally call the statistics
+     * function to get motion vectors and statistics like variances, distortions before
+     * calling Encode function via this entry point.
+     *
+     * Checking whether Statistics is supported can be performed with vaQueryConfigEntrypoints()
+     * and the profile argument set to AVC profiles. If Statistics entry point is supported,
```

Comment:
Does this function support AVC only?

Reply:
Currently yes, we only support VAEntrypointStats of AVC, and HEVC FEI will reuse VAEntrypointStats of AVC for saving big effort on VAEntrypointStats of HEVC.
For long term, we will add support to VAEntrypointStats of HEVC.

Reply:
Since "VAEntrypointStats" is a generic entrypoint, we shouldn't put the comment like "profile argument set to AVC profile".
Profile argument can be any profile and if the driver doesn't have support for that profile,  simply don't return the Stats entrypoint in vaQueryConfigEntrypoint().  Am I missing something?

Reply:
Agree with Sree, the API should be generic, we shouldn't add such requirement in API definition to workaround driver limitation.

Reply:
At beginning, we use NonProfile for we want it no profile limitation, but you suggest it to be AVC profile since actually driver internal works as AVC Profile, so we change it, driver always requires a profile input, I still think it better to be Non profile than any profiles here, how do you think about it?